src/test/java/core/basesyntax/operations/AddOperationTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.operations;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AddOperationTest {
+    private static final Fruit MANGO = new Fruit("mango");
+    private static final Integer INCREASE_BY_MANGO = 20;
+    private static final Fruit CHERRY = new Fruit("cherry");
+    private static final Integer CHERRY_QUANTITY = 150;
+    private static final Fruit BANANA = new Fruit("banana");
+    private static final Integer BANANA_QUANTITY = 50;
+    private static final Fruit BLUEBERRY = new Fruit("blueberry");
+    private static final Integer BLUEBERRY_QUANTITY = 23;
+    private static final Integer INCREASE_BY_CHERRY = 45;
+    private static final Integer INCREASE_BY_BANANA = 55;
+    private static final Integer INCREASE_BY_BLUEBERRY = 425;
+    private static Operation operation;
+    private Map<Fruit, Integer> expected;
+    private Map<Fruit, Integer> actual;
```

Comment:
make them local variables  as you did in the FruitDaoImplTest

Reply:
done